actions/index.js

```diff
@@ -1,3 +1,4 @@
+// Action for the currentFlies Reducer
```

Comment:
Thanks for adding in these comments! It definitely helps to make it more clear as to which items are doing what.